public/_redirects

```diff
@@ -216,5 +216,12 @@ https://docs.cert-manager.io/* https://cert-manager.io/docs/:splat 302!
 /docs/release-notes/* /docs/releases/release-notes/:splat 301!
 /docs/installation/supported-releases/ /docs/releases/ 301!
 
+## The /docs/installation/upgrading/ and /docs/release-notes/ pages do not exist anymore
+## however the catch-all redirect will redirect to the /docs/releases/upgrading/ and
+## /docs/releases/release-notes/ pages respectively. We will do an additional redirect
+## to the /docs/releases/ page to ensure that the user is redirected to an existing page.
+/docs/releases/upgrading/ /docs/releases/ 301!
```

Comment:
This causes a double redirect. Are you happy about that?

## Current

```bash
$ curl -IL https://cert-manager.io/docs/installation/upgrading/
HTTP/2 301
age: 64170
cache-control: public,max-age=0,must-revalidate
content-type: text/plain; charset=utf-8
date: Tue, 24 Oct 2023 08:19:23 GMT
location: /docs/releases/upgrading/
server: Netlify
strict-transport-security: max-age=31536000
x-nf-request-id: 01HDGC8QJ2V6NW73QKKF46XVS0
content-length: 40

HTTP/2 404
age: 712
cache-control: public,max-age=0,must-revalidate
content-type: text/html; charset=utf-8
date: Tue, 24 Oct 2023 08:19:23 GMT
etag: 1570710592-ssl
server: Netlify
strict-transport-security: max-age=31536000
x-nf-request-id: 01HDGC8QJT5MZJKC72DFD7ZZ4X
content-length: 39042
```

## New

```bash
$ curl -IL https://deploy-preview-1332--cert-manager-website.netlify.app/docs/installation/upgrading/
HTTP/2 301
age: 1
cache-control: public,max-age=0,must-revalidate
content-type: text/plain; charset=utf-8
date: Tue, 24 Oct 2023 08:20:15 GMT
location: /docs/releases/upgrading/
server: Netlify
strict-transport-security: max-age=31536000; includeSubDomains; preload
x-nf-request-id: 01HDGCA9CPEXC8HCP9S1Y9J1XE
x-robots-tag: noindex
content-length: 40

HTTP/2 301
age: 0
cache-control: public,max-age=0,must-revalidate
content-type: text/plain; charset=utf-8
date: Tue, 24 Oct 2023 08:20:15 GMT
location: /docs/releases/
server: Netlify
strict-transport-security: max-age=31536000; includeSubDomains; preload
x-nf-request-id: 01HDGCA9NPCSQCGFNZ8CPQ5FZP
x-robots-tag: noindex
content-length: 30

HTTP/2 200
accept-ranges: bytes
age: 0
cache-control: public,max-age=0,must-revalidate
content-type: text/html; charset=UTF-8
date: Tue, 24 Oct 2023 08:20:15 GMT
etag: "0a8e83c3cfee3231fe32800bd1149117-ssl"
server: Netlify
strict-transport-security: max-age=31536000; includeSubDomains; preload
x-nf-request-id: 01HDGCA9W4RJA7ZM9SWH1GWCK6
x-robots-tag: noindex
content-length: 187677

```

Reply:
I think we have to do a double redirect, right? the catch all above also catches this url.